integration-tests/trait-dyn-cross-contract-calls/contracts/incrementer/lib.rs

```diff
@@ -0,0 +1,60 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+#![allow(clippy::new_without_default)]
+
+#[ink::contract]
+pub mod incrementer {
+    use dyn_traits::Increment;
+
+    /// A concrete incrementer smart contract.
+    #[ink(storage)]
+    pub struct Incrementer {
+        value: u64,
+    }
+
+    impl Incrementer {
+        /// Creates a new incrementer smart contract initialized with zero.
+        #[ink(constructor)]
+        pub fn new(init_value: u64) -> Self {
```

Comment:
No point in making this configurable, just set it to `u64::default()` in the constructor